CraftsmenLtd/coe-batch-q322
# 170 step 1 dipto terraform initialization with lambda

Fixes #170 


## What kind of change does this PR introduce (check at least one)


## How would you describe the changes in a changelog (1-2 lines)?
A new lambda function has been created with terraform.

## What was done
A new lambda with source zip code has been initialized with terraform. Iam role for the terraform also has been created.

## How to test
After doing terraform apply, need to check whether tha lambda has been created or not.

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.


## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified
